Rebrand from Zing.gg to Cardcorp

VrandonB opened the pull request:
Updated any mentions of Zing or Zing.gg to Cardcorp in the plugin.

VrandonB committed:
Update README.md

Change branding from Zing.gg to Cardcorp, fix broken image link at the footer

VrandonB committed:
Update SVG logo and rename file to Cardcorp branding

VrandonB committed:
Rebrand error_list.php

VrandonB committed:
Update stylesheet with Cardcorp branding

VrandonB committed:
Update and rename zing_additional.php

VrandonB committed:
Update en_US.po with Cardcorp branding

VrandonB committed:
Update custom.js

VrandonB committed:
Update custom_rs

VrandonB committed:
Fix spelling error in stylesheet

VrandonB committed:
Update woocommerce-gateway-zing.php

VrandonB committed:
Rename  woocommerce-gateway-zing

VrandonB committed:
Remove metadata files and add .gitignore

VrandonB committed:
Fix broken link in widget logo

RTJ commented on README.md:
```diff
@@ -1,13 +1,15 @@
-[![](https://zing.gg/packs/media/images/public_pages/brand/zing-gg-dark-dd6352061e7352dc10446cb557312946.svg)](https://www.zing.gg "![](https://zing.gg/packs/media/images/public_pages/brand/zing-gg-dark-dd6352061e7352dc10446cb557312946.svg)")
+<div align="center">
+    <a href="https://www.cardcorp.com/" >
+        <img src="https://cardcorp.com/wp-content/uploads/2024/08/Logo_Cardcorp.svg" width="200" >
```
Image URL Usage

Do not use direct website URLs:
Avoid referencing images directly from our website (e.g., https://cardcorp.com/wp-content/uploads/2024/08/Logo_Cardcorp.svg). These images can be updated, replaced, or removed at any time.

Use CDN links:
Ensure that all image references in the plugin use CDN links for consistency and reliability.

Asset Preparation:
Please prepare and list all the images/assets you need. I will handle uploading them to the CDN once finalized.

VrandonB replied:
Updated in latest commits:
-  Added a function on plugin activation which checks the WooCommerce database if there's any tokens with a gateway ID of "zing". If yes: update to "cardcorp", if no: proceed as usual. Tests were successful, tokens that were previously labeled as zing were updated to cardcorp and are accessible after upgrading the plugin.
- Image source is now a local file;

RTJ commented on assets/css/cardcorp-style.css:
```diff
@@ -0,0 +1,532 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto);
+div.form-head {
```
Custom CSS for the Plugin

Avoid Generic Names:
Do not use generic CSS selectors or class names, as they may override client CSS and lead to conflicts.

Use Plugin-Specific Naming:
All CSS classes/tags must be uniquely identifiable to our plugin. For example, prefix class names with something like cardcorp_pay (e.g., cardcorp_pay-button, cardcorp_pay-container).

Consistency is Key:
Maintain a consistent naming convention throughout the plugin to ensure easy maintenance and to avoid any unintended style overrides.

VrandonB replied:
Updated in the latest commit. Removed the div.form-head selector entirely. I reviewed the code + the WordPress site, the .form-head is not an existing class.

RTJ reviewed (changes requested):
Testing Required:
Please test the migration on a staging environment to confirm that every customer’s card details remain intact after the plugin switch.

VrandonB committed:
Update image src in README.md to use a local asset

VrandonB committed:
Add functionality to update tokens with a Zing gateway ID to Cardcorp on plugin activation

VrandonB committed:
Remove .form-head class from stylesheet since it's unused

VrandonB committed:
Fix invalid header bug on plugin activation

Having multiple header metadata in separate files causes WooCommerce to throw an error

VrandonB committed:
Update version number to 1.6.0
